recipes/bcftools-snvphyl-plugin/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version="1.5" %}
+about:
+  home: https://github.com/phac-nml/snvphyl-tools
+  license: apache_2.0
+  summary: The SNVPhyl (Single Nucleotide Variant PHYLogenomics) pipeline is a pipeline for identifying Single Nucleotide Variants (SNV) within a collection\
+   of microbial genomes and constructing a phylogenetic tree. This package is the bcftools C plugin
+
+build:
+  number: 1
```

Comment:
0

Reply:
Fixed. Second time now that I forgot to change it to 0.